apps/health_campaign_field_worker_app/lib/utils/I18KeyConstants.dart

```diff
@@ -0,0 +1,23 @@
+class i18 {
+  static Common common = const Common();
+  static SearchBeneficiary searchBeneficiary = const SearchBeneficiary();
+}
+
+class Common {
+  const Common();
+  String get coreCommonContinue => 'CORE_COMMON_CONTINUE';
+  String get coreCommonAge => 'CORE_COMMON_AGE';
+  String get coreCommonGender => 'CORE_COMMOM_GENDER';
+}
+
+class SearchBeneficiary {
+  const SearchBeneficiary();
+  String get listOfHouseholds => 'LIST_OF_HOUSE_HOLDS';
+  String get noOfHouseholdsRegistered => 'NO_OF_HOUSE_HOLDS_REGISTERED';
+  String get noOfBedetsDelivered => 'NO_OF_BEDETSDELIVERED';
+  String get searchHintText => 'SEARCH_HINT_TEXT';
```

Comment:
Is this for the search by name of head of household search? We should have a more specific key if so.